examples/drivers/windows/otLwf/command.c

```diff
@@ -127,6 +134,12 @@ otLwfCmdInitialize(
             break;
         }
 
+        NtStatus = otLwfCmdResetDevice(pFilter, FALSE);
+        if (!NT_SUCCESS(NtStatus))
+        {
+            Status = NDIS_STATUS_FAILURE;
+        }
+
     } while (FALSE);
```

Comment:
This seems like an abuse of the ``do{ ... } while()`` loop, since it is not intended to loop at all.  Would this not be a job better suited to a `goto` statement, since it doesn't imply to the reader that you're going to loop?  Infact, using ``break`` is essentially the same as using ``goto`` in this situation.